### PR TITLE
Add support for ELBv2 (Application Loadbalancer)

### LINKED ISSUE #50: Support Application Load Balancer (ELBv2)
Required to support https://github.com/zalando-stups/senza/issues/310

### PR DESCRIPTION
**Pull request by mohabusama**
- Add `elb_type` - classic vs application
- Both elbs are considered entity with type `elb`

Resolve #50

### CODE REVIEW

**Comment by hjacobs** [comment]
Don't we need additionally the LB ARN now? (e.g. to get CloudWatch metrics)

**Comment by mohabusama** [comment]
Right! totally forgot about them 👀

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -360,10 +360,13 @@ def get_running_elbs_application(region, acc):
             'elb_type': 'application',
             'dns_name': e['DNSName'],
             'host': e['DNSName'],
-            'name': name,
+            'name': '/'.join(arn.rsplit('/')[-3:]),  # name used by Cloudwatch!
```
Comment (hjacobs) [inline zmon_aws_agent/aws.py]: hmm, I would keep the name as it was actually (the fact that we need to use the last parts for CloudWatch should not have any effect here)

File: zmon_aws_agent/aws.py [hunk 2]
```diff
@@ -360,10 +360,13 @@ def get_running_elbs_application(region, acc):
...
+            'elb_arn': arn,
+            'elb_name': name,
```
Comment (hjacobs) [inline zmon_aws_agent/aws.py]: if `name` is the ELB name (again), we can remove `elb_name`

File: zmon_aws_agent/aws.py [hunk 3]
```diff
@@ -320,6 +325,78 @@ def get_running_elbs(region, acc):
...
+            'id': entity_id('elb-{}[{}:{}]'.format(name, acc, region)),
+            'type': 'elb',
+            'infrastructure_account': acc,
+            'region': region,
+            'created_by': 'agent',
+            'elb_type': 'application',
+            'dns_name': e['DNSName'],
+            'host': e['DNSName'],
+            'name': '/'.join(arn.rsplit('/')[-3:]),  # name used by Cloudwatch!
+            'elb_arn': arn,
```
Comment (hjacobs) [inline zmon_aws_agent/aws.py]: maybe just `arn`? so we can reuse the field name for other entities (all AWS resources have ARNs)

File: zmon_aws_agent/aws.py (hunk 1 above)
Comment (mohabusama) [inline zmon_aws_agent/aws.py]: I noticed that Cloudwatch checks use `entity['name']` so this is why I adjusted the name here, while keeping the original as `elb_name`.

File: zmon_aws_agent/aws.py (hunk 3 above)
Comment (mohabusama) [inline zmon_aws_agent/aws.py]: Ok. Should I adjust all other entities as well with `arn` or only elbv2 for now?

File: zmon_aws_agent/aws.py (hunk 1 above)
Comment (mohabusama) [inline zmon_aws_agent/aws.py]: Will revert back to `name` and add a new field `cloudwatch_name`.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**